src/js/pages/system/UnitsHealthNodeDetail.js

```diff
@@ -33,12 +43,22 @@ class UnitsHealthNodeDetail extends mixin(StoreMixin) {
     UnitHealthStore.fetchUnitNode(unitID, unitNodeID);
   }
 
-  getErrorNotice() {
-    return (
-      <div className="container container-pod">
-        <RequestErrorMsg />
-      </div>
-    );
+  onUnitHealthStoreUnitSuccess() {
+    let isLoading = this.state.isLoading & ~FLAGS.UNIT_LOADED;
```

Comment:
Yeah, they'll take some getting used to. But it's really nice for components that depend on more than one request before rendering. We could even tie in the error as a flag too. That would be nice that way we don't have another variable tracking that.